### PR TITLE
added initial settings, displayed product list

### PR DESCRIPTION
**Pull request by Ksenaaa**

### CODE REVIEW

File: src/components/customButton/CustomButton.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+import { Colors } from 'utils/constants/Colors';
+
+interface Props {
+    title: string;
+    onPress: () => void;
+    disabled?: boolean;
+}
+
+export const CustomButton = ({ title, onPress, disabled }: Props) => (
```
Comment (vell28) [inline src/components/customButton/CustomButton.tsx]: `{ title, onPress, disabled }` to `props`

File: src/components/customButton/CustomButton.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <View style={styles.container}>
+        <Button
```
Comment (vell28) [inline src/components/customButton/CustomButton.tsx]: and than {...props}

File: src/components/customIcon/CustomIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+import Feather from 'react-native-vector-icons/Feather';
+
+interface Props {
+    name: string;
+    size: number;
+    color: number | ColorValue | undefined;
+    disabled?: boolean;
+}
+
+export const CustomIcon = ({ name, size, color, disabled }: Props) => {
```
Comment (vell28) [inline src/components/customIcon/CustomIcon.tsx]: here the same

File: src/navigators/TabNavigator.tsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        >
+            <Tab.Screen
+                name="Home"
+                component={HomeScreen}
+                options={{
+                    tabBarIcon: ({ focused, color, size }) => (
+                        <CustomIcon
+                            name="home"
+                            size={25}
+                            color={focused ? Colors.gold : Colors.grey}
```
Comment (vell28) [inline src/navigators/TabNavigator.tsx]: move this logic to function and use everywhere

File: src/services/app.ts [hunk 5]
```diff
@@ -0,0 +1,56 @@
+const baseURL = process.env.REACT_APP_DEV_MODE ?? '';
```
Comment (vell28) [inline src/services/app.ts]: rename to api.ts

File: src/utils/helpers/productColors.ts [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import { ProductMaterial } from 'model/productMaterial'
+
+export const productColors = (colorsProduct: string[], colors: ProductMaterial[]) => {
+    const dataColors = [] as ProductMaterial[]
+
+    colorsProduct.forEach(id => {
```
Comment (vell28) [inline src/utils/helpers/productColors.ts]: rewrite to filter

File: src/components/customButton/CustomButton.tsx (hunk 2 above)
Comment (Ksenaaa) [reply src/components/customButton/CustomButton.tsx]: fixed

File: src/components/customIcon/CustomIcon.tsx (hunk 3 above)
Comment (Ksenaaa) [reply src/components/customIcon/CustomIcon.tsx]: fixed

File: src/components/customButton/CustomButton.tsx (hunk 1 above)
Comment (Ksenaaa) [reply src/components/customButton/CustomButton.tsx]: fixed

File: src/navigators/TabNavigator.tsx (hunk 4 above)
Comment (Ksenaaa) [reply src/navigators/TabNavigator.tsx]: fixed

File: src/services/app.ts (hunk 5 above)
Comment (Ksenaaa) [reply src/services/app.ts]: fixed

File: src/utils/helpers/productColors.ts (hunk 6 above)
Comment (Ksenaaa) [reply src/utils/helpers/productColors.ts]: fixed

**Pull request merged**